AppHub/AppHub/AHBuild.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithBundle:(NSBundle *)bundle info:(NSDictionary *)info
     if ((self = [super init])) {
         _buildDescription = [info[AHBuildDataDescriptionKey] copy] ?: @"This build was downloaded from the App Store.";
         _bundle = bundle ?: [NSBundle mainBundle];
-        _compatibleIOSVersions = info[AHBuildDataCompatibleIOSVersionsKey] ?: AHShortVersionString();
+        _compatibleIOSVersions = [info[AHBuildDataCompatibleIOSVersionsKey] allValues] ?: @[AHShortVersionString()];
```

Comment:
Got it, I'll take into account the conversation here, cleanup the commits and then resubmit the PR.